scripts/novaseq/demux-novaseq.bash

```diff
@@ -41,7 +41,7 @@ ${BCL2FASTQ_BIN} --version
 # Here we go!
 log "Here we go!"
```

Comment:
Jolly! But is there a more informative log message to communicate?

Reply:
This is Kenny legacy :) 

I'll replace it with something more useful.